src/AJT/Toggl/services_v9.json

```diff
@@ -115,7 +179,7 @@
     },
     "GetClients": {
       "httpMethod": "GET",
-      "uri": "clients",
+      "uri": "me/clients",
       "summary": "Get clients visible to user",
       "responseModel": "getResponse"
     },
```

Comment:
GetClientProjects does not seem to exist anymore. But i don't see it in the migration notes either. Should this be workspace?

Reply:
@arendjantetteroo workspaces and clients are 2 different concepts. This endpoint now also requires the workspace ID parameters, just like the POST request to update a client or GET to get a single client.

The URI should be  `workspaces/{workspace_id}/clients`

And the parameters 
`"parameters": {
        "workspace_id": {
          "location": "uri",
          "type": "integer",
          "required": true,
          "description": "Workspace ID"
        },`

I'm trying to edit this PR or add code changes directly to this review but I'm not able to. Probably a rights issue, which seem fair enough.

Reply:
It is the next item in the json, not related to getClients :) 

I think you can just pull this branch locally, create your own and then push to your own fork and make a new PR.

Reply:
@arendjantetteroo, my bad; I looked at where you put the comment and looked up the calls to get clients and assumed that was what you meant, but I read over  "GetClientProjects" for some reason. What I put above is how to get clients by workspace, which is GetWorkspaceProjects

As far as I see you are right, GetClientProjects does no longer exists.